### ISSUE TITLE
Vault capture with blank token results in failure

### ISSUE THREAD

**Issue by davemay99** [hashicorp/hcdiag#88]
When capturing from a Vault server without a token, `hcdiag` prints an error message and exits.  Running the underlying `vault debug` works and does not require a VAULT_TOKEN environment variable or `~/.vault_token` file.

hcdiag output:
```
dmay@ubuntu:~$ hcdiag -version
hcdiag v0.2.0

dmay@ubuntu:~$ hcdiag -vault
2022-04-04T18:54:57.927-0400 [INFO]  hcdiag: Checking product availability
2022-04-04T18:54:58.020-0400 [ERROR] hcdiag: Failed running Products: error="unable to find VAULT_TOKEN env or ~/.vault-token"
```

vault debug output:
```
dmay@ubuntu:~$ vault debug
==> Starting debug capture...
         Vault Address: http://192.168.1.64:8200
        Client Version: 1.10.0
              Duration: 2m0s
              Interval: 30s
      Metrics Interval: 10s
               Targets: config, host, requests, metrics, pprof, replication-status, server-status, log
                Output: vault-debug-2022-04-04T22-49-50Z.tar.gz

==> Capturing static information...
2022-04-04T18:49:50.364-0400 [INFO]  capturing configuration state
2022-04-04T18:49:50.368-0400 [ERROR] config: error capturing config state:
  error=
  | Error making API request.
  | 
  | URL: GET http://192.168.1.64:8200/v1/sys/config/state/sanitized
  | Code: 400. Errors:
  | 
  | * missing client token
  

==> Capturing dynamic information...
2022-04-04T18:49:50.368-0400 [INFO]  capturing metrics: count=0
2022-04-04T18:49:50.368-0400 [INFO]  capturing host information: count=0
2022-04-04T18:49:50.368-0400 [INFO]  capturing in-flight request status: count=0
2022-04-04T18:49:50.368-0400 [INFO]  capturing pprof data: count=0
2022-04-04T18:49:50.368-0400 [INFO]  capturing replication status: count=0
2022-04-04T18:49:50.368-0400 [INFO]  capturing server status: count=0
Finished capturing information, bundling files...
Success! Bundle written to: vault-debug-2022-04-04T22-49-50Z.tar.gz
```

Note that there is an error requiring a token for the state, but the bundle is written either way.

**Comment by davemay99** [comment]
The same results occur when either VAULT_TOKEN or `~/.vault_token` is empty/missing

**Comment by aliciaclark1066** [comment]
Hi @davemay99, is this still an issue? If so, we'll add it to our backlog.

**Comment by aliciaclark1066** [comment]
@davemay99 We've resolved this issue and will push the fix as part of v0.5.0.
